src/App.js

```diff
@@ -0,0 +1,43 @@
+import React, { useState, useEffect } from 'react';
+import { BrowserRouter, Switch, Route } from 'react-router-dom';
+import Dashboard from './components/dashboard/Dashboard.jsx';
+import Orders from './components/orders/order/Orders';
+import SignIn from './components/auth/SignIn';
+import AllOrders from './components/admon/all-orders/AllOrders';
+import Admin from './components/admon/Admin';
+import BurgerQueen from './components/BQ/BurgerQueen.jsx';
+import 'materialize-css/dist/css/materialize.min.css';
+import { firebase } from './firebase/firebaseConfig';
+
```

Comment:
Primero dependencias después componentes
```suggestion
import React, { useState, useEffect } from 'react';
import 'materialize-css/dist/css/materialize.min.css';
import { firebase } from './firebase/firebaseConfig';
import { BrowserRouter, Switch, Route } from 'react-router-dom';

import Dashboard from './components/dashboard/Dashboard.jsx';
import Orders from './components/orders/order/Orders';
import SignIn from './components/auth/SignIn';
import AllOrders from './components/admon/all-orders/AllOrders';
import Admin from './components/admon/Admin';
import BurgerQueen from './components/BQ/BurgerQueen.jsx';


```